src/app/settings/settings.service.ts

```diff
@@ -57,6 +64,9 @@ export class SettingsService {
    * Returns language setting
    */
   get language() {
+    if (!localStorage.getItem('mifosXLanguage')) {
```

Comment:
I assume this is a bugfix but then why didn't you create a separate PR?

Reply:
Related to the Language fix to start the support of i18n in this application